packages/atomic/create-atomic/src/plopfile.ts

```diff
@@ -51,72 +82,130 @@ export default function (plop: NodePlopAPI) {
+        name: 'page-id',
+        default: undefined,
+        loop: false,
+        pageSize: 10,
+        message:
+          'Use an existing hosted search page as a template, or start from scratch?',
+        choices: async function (answers: PlopData) {
+          initPlatformClient(answers);
+          return [
+            {value: undefined, name: 'Start from scratch'},
+            {type: 'separator'},
+            ...(await listSearchPagesOptions(platformClient)),
+          ];
+        },
+        validate: async (input, answers: PlopData) => {
+          try {
+            initPlatformClient(answers);
+            answers.page = await fetchPageManifest(platformClient, input);
 
-          process.on('close', (code) => {
-            if (code === 0) {
-              resolve('Installation complete');
-            } else {
-              reject(`Installation exited with ${code}`);
-            }
+            return true;
+          } catch (error) {
+            return `The search page with the id "${input}" does not exist.`;
+          }
+        },
+      } as PromptQuestion,
+    ],
+    actions: function () {
+      return [
+        function pageInfo(data) {
+          const plopData = data as PlopData;
+          if (plopData.page) {
+            return `Hosted search page "${plopData.page.config.title} has been downloaded`;
+          }
+
+          plopData.page = defaultPageManifest;
+          return 'Using the default search page template.';
+        },
+        {
+          type: 'add',
+          path: currentPath + '/{{project}}/.env',
+          templateFile: '../template/.env.hbs',
+        },
+        {
+          type: 'add',
+          path: currentPath + '/{{project}}/.gitignore',
+          templateFile: '../template/.gitignore.hbs',
+        },
+        {
+          type: 'add',
+          path: currentPath + '/{{project}}/package.json',
+          templateFile: '../template/package.json.hbs',
+        },
+        {
+          type: 'addMany',
+          destination: currentPath + '/{{project}}/',
+          base: '../template',
+          templateFiles: [
+            '../template/src/**',
+            '../template/scripts/*',
+            '../template/.env.example',
+            '../template/tsconfig.json',
+            '../template/stencil.config.ts',
+            '../template/netlify.toml',
+            '../template/README.md',
+            '../template/start-netlify.mjs',
+          ],
+        },
+        function generateTemplates(data) {
+          const {page, project} = data as PlopData;
+          page.results.templates.forEach((resultTemplate, index) => {
+            const filePath = join(
+              currentPath,
+              project,
+              'src',
+              'components',
+              'results-manager',
+              `template-${index + 1}.html`
+            );
+            writeFileSync(
+              filePath,
+              plop.renderString('{{{markup}}}', resultTemplate)
```

Comment:
Use prettier here ;)
1. Move prettier from `devDependencies` to `dependencies`
2. `import {format} from 'prettier';` somewhere uptop this file.
3.
```suggestion
              format(plop.renderString('{{{markup}}}', resultTemplate), {parser:'html'}
```